src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
@mattiaerre I'll dig into this. We're using ES6 imports, but usually in the style `import { Model } from '@graphistry/falcor'`. Though since the library is built with CommonJS and bundled with webpack, I'm not sure why default import style wouldn't work.